r/src/compute.cpp

```diff
@@ -288,6 +288,19 @@ std::shared_ptr<arrow::compute::FunctionOptions> make_compute_options(
         cpp11::as_cpp<arrow::TimeUnit::type>(options["unit"]));
   }
 
+  if (func_name == "strftime") {
+    using Options = arrow::compute::StrftimeOptions;
+    std::string format = "%Y-%m-%dT%H:%M:%SZ";
+    std::string locale = "C";
+    if (!Rf_isNull(options["format"])) {
+      format = cpp11::as_cpp<std::string>(options["format"]);
+    }
+    if (!Rf_isNull(options["locale"])) {
+      locale = cpp11::as_cpp<std::string>(options["locale"]);
+    }
+    return std::make_shared<Options>(Options(format, locale));
+  }
+
```

Comment:
Thanks for adding this, but since there aren't any tests that exercise this I don't think we're done with the R bindings. I've made ARROW-13448 for that.

Reply:
Removed and added a comment to ARROW-13448.